src/app/ubs-admin/components/ubs-admin-employee/ubs-admin-employee-card/ubs-admin-employee-card.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="main-card-container">
+  <div class="card-container">
+    <div class="employee-info">
+      <div *ngIf="data.image">
+        <img [src]="data.image" alt="image" />
```

Comment:
add correct alt tag

Reply:
photo of an employee. It will be better.